tests/test_modeling_common.py

```diff
@@ -2476,6 +2477,55 @@ def test_checkpoint_sharding_from_hub(self):
         for p1, p2 in zip(model.parameters(), ref_model.parameters()):
             self.assertTrue(torch.allclose(p1, p2))
 
+    def test_from_pretrained_low_cpu_mem_usage_functional(self):
+        # test that we can use `from_pretrained(..., low_cpu_mem_usage=True)` with normal and
+        # sharded models
+
+        mnames = [
+            "hf-internal-testing/tiny-random-bert-sharded",
+            "hf-internal-testing/tiny-random-bert",
+        ]
+        for mname in mnames:
+            _ = BertModel.from_pretrained(mname, low_cpu_mem_usage=True)
+
+    # @slow
+    # XXX: when switching back to slow, move `apt install time` to the github action running this test
+    @require_usr_bin_time
+    def test_from_pretrained_low_cpu_mem_usage_measured(self):
+        # test that `from_pretrained(..., low_cpu_mem_usage=True)` uses less cpu memory than default
+
+        mname = "bert-base-uncased"
```

Comment:
Do we have a normal non-tiny model that we use during non-slow tests? if it's pre-downloaded then this test is really fast.

Reply:
@LysandreJik?

Reply:
Other than `bert-base-cased`? Because yes, `bert-base-cased` is indeed used in other non-slow tests, so it will already be downloaded.

Reply:
perfect, I will use `bert-base-cased` then and won't use `@slow` as it's really fast.